tests/tests.rs

```diff
@@ -31,20 +31,27 @@ fn compare_generated_header(header: &PathBuf,
     if !expected.is_file() {
         let file_name = expected.file_name().unwrap().to_owned();
         expected.pop();
-
-        if cfg!(feature = "testing_only_libclang_4") {
-            expected.push("libclang-4");
-        } else if cfg!(feature = "testing_only_libclang_3_9") {
-            expected.push("libclang-3.9");
-        } else if cfg!(feature = "testing_only_libclang_3_8") {
-            expected.push("libclang-3.8");
+        
+        let version = clang_version();
+        match version.parsed {
+            None => panic!("Couldn't parse libclang version!"),
+            Some(version) => {
+                let (maj, min) = version;
+                if maj == 4 {
```

Comment:
Also, this could be organised as:

```rust
let version_str = if maj >= 4 {
    "4".to_owned()
} else {
    format!("{}.{}", maj, min)
};

expected.push(format!("libclang-{}", version_str))
```

to reduce duplication.

What do you think?